app/controllers/request-to-go-live/organisation-address/post.controller.js

```diff
@@ -113,22 +125,30 @@ const validateForm = function validate (form, isRequestToGoLive) {
   return orderedErrors
 }
 
-const submitForm = async function (form, serviceExternalId, correlationId, isRequestToGoLive) {
+const submitForm = async function (form, serviceExternalId, correlationId, isRequestToGoLive, isStripeUpdateOrgDetails) {
   const updateRequest = new ServiceUpdateRequest()
```

Comment:
Can `updateRequest` declaration be moved to else block ?

Reply:
Good idea - done